src/@next/Carousel/Carousel.tsx

```diff
@@ -138,7 +136,7 @@ export const Carousel = ({
           </NextArrow>
         </ArrowsContainer>
       )}
-      {showIndicator && (
+      {indicatorType != null && (
```

Comment:
probably we can check if it's falsy coz since it's optional, devs can pass undefined as well